frontend/src/components/CartItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Button, ListItem, ListItemText } from "@mui/material";
+
+interface CartItemProps {
+  id: string;
+  name: string;
+  price: number;
+  quantity: number;
+  removeFromCart: (id: string) => void;
+}
+
+const CartItem: React.FC<CartItemProps> = ({
+  id,
+  name,
+  price,
+  quantity,
+  removeFromCart,
+}) => {
+  return (
+    <ListItem
+      sx={{
+        marginBottom: "16px",
+        display: "flex",
+        justifyContent: "space-between",
+      }}
+    >
+      <ListItemText
+        style={{ color: "black" }}
+        primary={`${name} x${quantity}`}
+        secondary={`Cena: $${price.toFixed(2)} x${quantity}`}
+      />
+      <Button
+        variant="outlined"
+        color="secondary"
+        onClick={() => removeFromCart(id)}
+        style={{ color: "black" }}
+        sx={{
+          border: "2px solid black",
+          "&:hover": {
+            backgroundColor: "red",
+            borderColor: "red",
+          },
+          color: "black",
+        }}
+        
+      >
+        Usuń
```

Comment:
pleause use english langauge in whole app

Reply:
Nie rzucim ziemi, skąd nasz ród,
Nie damy pogrześć mowy!
Polski my naród, polski lud,
Królewski szczep Piastowy.
Nie damy, by nas gnębił wróg!

Tak nam dopomóż Bóg!


Do krwi ostatniej kropli z żył
Bronić będziemy ducha,
Aż się rozpadnie w proch i pył
Krzyżacka zawierucha.
Twierdzą nam będzie każdy próg,

Tak nam dopomóż Bóg!


Nie będzie Niemiec pluł nam w twarz,
Ni dzieci nam germanił,
Orężny wstanie hufiec nasz,
Duch będzie nam hetmanił.
Pójdziem, gdy zabrzmi złoty róg,

Tak nam dopomóż Bóg!